lang/cs/Org.Apache.REEF.Utilities/Logging/Logger.cs

```diff
@@ -47,7 +47,9 @@ public sealed class Logger
                         { Level.Verbose, TraceEventType.Verbose },
                     };
 
-        private static Level _customLevel = Level.Info;
+        private static Level _globalLevel = Level.Info;
+
+        private Level _customLevel = Level.Unset;
```

Comment:
I would call this field `_instanceLevel`, to make sure we get away from old meaning of "custom level".